resources/monitoring/templates/prometheus-operator/prometheus-operator.rules.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Please move this file to `kyma-additions` folder to keep this folder as similar as possible to the upstream chart.